app/controllers/newflow/base_controller.rb

```diff
@@ -11,6 +12,19 @@ def profile_newflow
       render layout: 'application'
     end
 
+    # SheerID makes a POST request to this endpoint when it verifies an educator
+    # http://developer.sheerid.com/program-settings#webhooks
+    def sheerid_webhook
```

Comment:
you want this on the basecontroller, available to all controllers?  should it be off on its own?

Reply:
Sure, sounds like a good idea. Created an `OtherController`.